docs/docs-content/registries-and-packs/deploy-pack.md

```diff
@@ -595,62 +887,58 @@ In the **Nodes config** section, provide the details for the master and the work
+Review all configurations in this section. The **Review** page displays the cluster name, tags, cloud account name, node pools, layers, and an estimated hourly cost. If everything is correct, click on the **Finish Configuration** button to complete the cluster deployment, which may take up to *20 minutes*. 
 
-<br /> 
+While the deployment is in progress, Palette displays the cluster status as **Provisioning**. While you wait for the cluster to finish deploying, you can explore the various tabs on the cluster page, such as **Overview**, **Workloads**, and **Events**. 
 
-<!-- </TabItem>
+<br />
 
-<TabItem label="Terraform Workflow" value="terraform_code"> -->
+</TabItem>
 
-## Terraform Workflow
+<TabItem label="Terraform Workflow" value="Terraform_Workflow">
 
-The [Spectro Cloud Terraform provider](https://registry.terraform.io/providers/spectrocloud/spectrocloud/latest/docs) allows you to create and manage Palette resources using Infrastructure as Code (IaC). This offers such advantages as automating infrastructure, facilitating collaboration, documenting infrastructure, and keeping all infrastructure in a single source of truth.
+The [Spectro Cloud Terraform Provider](https://registry.terraform.io/providers/spectrocloud/spectrocloud/latest/docs) allows you to create and manage Palette resources using Infrastructure as Code (IaC). This approach offers several advantages, including automating infrastructure, facilitating collaboration, documenting infrastructure, and keeping a single source of truth for all infrastructure.
 
 ### Starter Code
-Navigate back to your tutorials container bash session to locate the starter Terraform files. If you have closed the terminal session, you can reopen another bash session in the tutorials container using the following command. 
-<br />
+
+Return to your tutorial container bash session to locate the starter Terraform files. If you have closed the terminal session, you can open another bash session in the tutorial container using the following command. 
 
 ```bash
 docker exec -it tutorialContainer bash
 ```
-
-Switch to the **/terraform/pack-tf** directory, which contains the Terraform code for this tutorial. 
 <br />
 
+Navigate to the **/terraform/pack-tf** directory, which contains the Terraform code for this tutorial. 
+
 ```bash
 cd /terraform/pack-tf
 ```
+<br />
 
 ### Set Up the Spectro Cloud API Key
 
-To get started with Terraform code, you need a Spectro Cloud API key to authenticate and interact with the Palette API endpoint. To add a new API key, log in to Palette, click on the user **User Menu** at the top right, and select **My API Keys**, as shown in the screenshot below. 
+To get started with Terraform code, you need a Spectro Cloud API key to authenticate and interact with the Palette API endpoint. To add a new API key, log in to Palette, click on the **User Menu** at the top right, and select **My API Keys**, as shown in the screenshot below. 
```

Comment:
```suggestion
To get started with Terraform code, you need a Palette API key to authenticate and interact with the Palette API endpoint. To add a new API key, log in to Palette, click on the **User Menu** at the top right, and select **My API Keys**, as shown in the screenshot below.  
```
Let's also link to the API Key how-to
https://docs.spectrocloud.com/user-management/authentication/api-key/create-api-key